dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/program/ProgramStageServiceTest.java

```diff
@@ -78,8 +80,13 @@ void testSaveProgramStage()
     {
         long idA = programStageService.saveProgramStage( stageA );
         long idB = programStageService.saveProgramStage( stageB );
+
         assertNotNull( programStageService.getProgramStage( idA ) );
         assertNotNull( programStageService.getProgramStage( idB ) );
+
+        ProgramStage persistedProgramStage = programStageService.getProgramStage( idA );
```

Comment:
Can you move these assertions up to so they are grouped with `assertNotNull( programStageService.getProgramStage( idA ) );`. Add a newline before `assertNotNull( programStageService.getProgramStage( idB ) );`